packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -99,11 +99,14 @@ export function getBeaconPoolApi({
     async submitPoolBlsToExecutionChange(blsToExecutionChanges) {
       const errors: Error[] = [];
 
+      // TODO CAPELLA: Which fork should assume to validate submitted execution changes?
+      const signatureFork = chain.config.getForkName(chain.clock.currentSlot);
```

Comment:
i think this would be correct fork to assume as we can assume that submitted message is ready to be included in the current slot